modules/custom/openy_map/src/Form/SettingsForm.php

```diff
@@ -68,6 +68,37 @@ public function buildForm(array $form, FormStateInterface $form_state) {
     $config = $this->config('openy_map.settings');
     $form_state->setCached(FALSE);
 
+    $form['map_engine_title'] = [
+      '#markup' => '<h2>' . $this->t('Map provider') . '</h2>',
+    ];
+    $form['map_engine'] = [
+      '#type' => 'radios',
+      '#options' => [
+        'leaflet' => $this->t('Leaflet + OpenStreetMaps tiles'),
```

Comment:
Let's call it "Leaflet" for now. Customers do not need to know about tiles for now. Later on we can add options list with tiles selection.